torchsde/_core/adjoint_sde.py

```diff
@@ -38,8 +38,8 @@ def __init__(self,
         }.get(sde.noise_type)
         super(AdjointSDE, self).__init__(sde_type=sde_type, noise_type=noise_type)
 
-        self._base_sde = sde
-        self._params = params
+        self.base_sde = sde
```

Comment:
Thinking now, we should've just named this `forward_sde` to avoid any further confusion. WDYT?